deployers/servicedeployer.go

```diff
@@ -119,7 +119,7 @@ func (deployer *ServiceDeployer) ConstructDeploymentPlan() error {
 
 	manifestReader.InitRootPackage(manifestParser, manifest)
 
-	if deployer.IsDefault == true && !utils.Flags.WithinOpenWhisk {
```

Comment:
I am curious as why we were testing "WithinOpenWhisk" here in the first place...

Reply:
With the new "export" capability added by @kpavel, the directory structure will be exported to the FS of the container, in which the action runs. The use case would be: "export" assets from one OW instance and "import" into another one via running wskdeploy from within an action.

Reply:
With the new "export" capability added by @kpavel, the directory structure will be exported to the FS of the container, in which the action runs. The use case would be: "export" assets from one OW instance and "import" into another one via running wskdeploy from within an action.